packages/migration/src/mods/7.0.0/colorPickerWarnRemovedProps.ts

```diff
@@ -9,7 +11,7 @@ const onChangedPropsByComponent: { [component: string]: string } = {
 };
 
 export default migration(
-  'warn ColorPicker-related onChanged props removed',
+  getWarningNote('ColorPicker onChanged prop was removed.'),
```

Comment:
I like the format of `ComponentName: ...` (with the `:`). Also I don't think the period at the end is necessary, but that's more nitpicky.
```suggestion
  getWarningNote('ColorPicker: onChanged prop was removed.'),
```

Reply:
Hmm actually I take both of those comments back now that I see how the warning is used in the message.

Reply:
Thinking about it more :P I wonder if it's better to omit the period here, then add it in `getWarningNote` when constructing the final message. That way there's guaranteed to be a period, whereas if you rely on the caller to provide a period, they may omit it, which results in a confusing message.